lib/youtube-dl.js

```diff
@@ -234,7 +234,9 @@ ytdl.getInfo = function(url, args, options, callback) {
 
 
 var formatsRegex =
-  /^(\d+)\s+([a-z0-9]+)\s+(\d+x\d+|\d+p|audio only|\?x\d+|unknown)/;
+  /^(\d+)\s+([a-z0-9]+)\s+(\d+x\d+|\d+p|audio only|\?x\d+|unknown)(.*)$/;
+var extendedRegex =
```

Comment:
or maybe it would be better to split the rest of the string by commas, and parse all of those fields not just the bitrate